auth/auth.go

```diff
@@ -128,15 +129,18 @@ func NewClient(ctx context.Context, c *internal.AuthConfig) (*Client, error) {
 		projectID: c.ProjectID,
 		snr:       snr,
 		url:       idToolKitURL,
+		version:   c.Version,
```

Comment:
Lets assign the formatted string `Go/Admin/<version>` here, so we don't have to recompute it for each request.